src/main/java/com/juniorstart/juniorstart/payload/ChangeMailRequest.java

```diff
@@ -5,28 +5,30 @@
 import jakarta.validation.constraints.NotBlank;
 import lombok.Getter;
 
+import java.util.UUID;
+
 /** Class for method changeEmail from UserService
  * @author Dawid Wit
- * @version 1.0
+ * @version 1.1
  * @since 1.0
  */
 @Getter
-public class ChangeMailRequest {
+public class ChangeMailRequest implements ChangeRequest{
     @Email
-    String email;
+    private final String email;
     @NotBlank
-    String name;
+    private final UUID privateId;
     @NotBlank
-    String password;
+    private final String password;
 
     /** Creates an object ot change email by UserService.changeEmail.
      * @param email new mail for account.
-     * @param name name user who change email.
+     * @param privateId of user who change email.
      * @param password for account.
      */
-    public ChangeMailRequest(@Email @Max(40) String email, @NotBlank String name, @NotBlank String password) {
+    public ChangeMailRequest(@Email @Max(40) String email, UUID privateId, @NotBlank String password) {
```

Comment:
Jeżeli adnotacje są przy polach, które inicjują zmienne to w konstruktorze nie są potrzebne.